chainio/clients/elcontracts/writer_test.go

```diff
@@ -1123,6 +1123,360 @@ func setTestRewardsCoordinatorActivationDelay(
+		claim, err := newTestClaim(chainReader, anvilHttpEndpoint, cumulativeEarnings, privateKeyHex)
+		require.NoError(t, err)
+
+		receipt, err = chainWriter.ProcessClaim(
+			context.Background(),
+			*claim,
+			anvilFirstAddr,
+			true,
+		)
+		assert.Error(t, err)
+		assert.Nil(t, receipt)
+	})
+
+	t.Run("process claims", func(t *testing.T) {
+		rewardsCoordinatorAddr := contractAddrs.RewardsCoordinator
+		config := elcontracts.Config{
+			DelegationManagerAddress:  contractAddrs.DelegationManager,
+			RewardsCoordinatorAddress: rewardsCoordinatorAddr,
+		}
+		chainReader, err := testclients.NewTestChainReaderFromConfig(anvilHttpEndpoint, config)
+		require.NoError(t, err)
+
+		activationDelay := uint32(0)
+		receipt, err := setTestRewardsCoordinatorActivationDelay(anvilHttpEndpoint, privateKeyHex, activationDelay)
+		require.NoError(t, err)
+		require.Equal(t, gethtypes.ReceiptStatusSuccessful, receipt.Status)
+
+		cumulativeEarnings := int64(42)
+		claim, err := newTestClaim(chainReader, anvilHttpEndpoint, cumulativeEarnings, privateKeyHex)
+		require.NoError(t, err)
+
+		receipt, err = chainWriter.ProcessClaims(
+			context.Background(),
+			[]rewardscoordinator.IRewardsCoordinatorTypesRewardsMerkleClaim{*claim},
+			anvilFirstAddr,
+			true,
+		)
+		assert.Error(t, err)
+		assert.Nil(t, receipt)
+	})
+
+	t.Run("set operator AVS split", func(t *testing.T) {
+		receipt, err := chainWriter.SetOperatorAVSSplit(
+			context.Background(),
+			common.HexToAddress(operatorAddr),
+			anvilFirstAddr,
+			uint16(1),
+			true,
+		)
+		assert.Error(t, err)
+		assert.Nil(t, receipt)
+	})
+
+	t.Run("set operator PI split", func(t *testing.T) {
+		receipt, err := chainWriter.SetOperatorPISplit(
+			context.Background(),
+			common.HexToAddress(operatorAddr),
+			uint16(1),
+			true,
+		)
+		assert.Error(t, err)
+		assert.Nil(t, receipt)
+	})
+
+	t.Run("modify allocations", func(t *testing.T) {
+		strategyAddr := contractAddrs.Erc20MockStrategy
+		avsAddr := anvilFirstAddr
+		operatorSetId := uint32(1)
+
+		operatorSet := allocationmanager.OperatorSet{
+			Avs: avsAddr,
+			Id:  operatorSetId,
+		}
+		newAllocation := uint64(100)
+		allocateParams := []allocationmanager.IAllocationManagerTypesAllocateParams{
+			{
+				OperatorSet:   operatorSet,
+				Strategies:    []common.Address{strategyAddr},
+				NewMagnitudes: []uint64{newAllocation},
+			},
+		}
+
+		receipt, err := chainWriter.ModifyAllocations(
+			context.Background(),
+			common.HexToAddress(operatorAddr),
+			allocateParams,
+			true,
+		)
+		assert.Error(t, err)
+		assert.Nil(t, receipt)
+	})
+
+	t.Run("set allocation delay", func(t *testing.T) {
+		receipt, err := chainWriter.SetAllocationDelay(
+			context.Background(),
+			common.HexToAddress(operatorAddr),
+			uint32(0),
+			true,
+		)
+		assert.Error(t, err)
+		assert.Nil(t, receipt)
+	})
+
+	t.Run("deregister from operator sets", func(t *testing.T) {
+		avsAddress := common.HexToAddress("0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266")
```

Comment:
We could use the constant value of the first Anvil address here.

```suggestion
		avsAddress := common.HexToAddress(testutils.ANVIL_FIRST_ADDRESS)
```

Reply:
Done in [Use anvil first address constant for avsAddress](https://github.com/Layr-Labs/eigensdk-go/pull/454/commits/a88a5c38f48a1ae2f07d8e213b422213d86d4797)